conf/modules.config

```diff
@@ -66,7 +68,12 @@ process {
     }
 
     withName: CUTADAPT_READTHROUGH {
-        ext.args = "--minimum-length 1 -a ${RV_primer_RevComp} -A ${FW_primer_RevComp}"
+        ext.args = [
+            "--minimum-length 1",
+            "-O ${params.cutadapt_min_overlap}",
+            "-e ${params.cutadapt_max_error_rate}",
+            "-a ${RV_primer_RevComp} -A ${FW_primer_RevComp}"
```

Comment:
Don't you want to give the parameter `-a` and `-A` a separate line?

Reply:
Possible, but those two are required to come in a pair, and thats not a bad thing when this is also visually clear here I think.